src/main/java/org/apache/maven/plugins/enforcer/RequireLicense.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.maven.plugins.enforcer;
+
+import org.apache.maven.enforcer.rule.api.EnforcerRule;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
+import org.apache.maven.model.License;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.component.configurator.expression.ExpressionEvaluationException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Manuel Recena
+ * @since TODO
+ */
+public class RequireLicense implements EnforcerRule {
+
+    public void execute(EnforcerRuleHelper helper) throws EnforcerRuleException {
+        Log log = helper.getLog();
+        try {
+            MavenProject project = (MavenProject) helper.evaluate( "${project}" );
```

Comment:
Null and classcast checks are missing here. Seems to be NIT in this case

Reply:
Agree. I did not know how to manager this because `helper.evaluate()` always returns `Object` and throws an exception if something goes wrong.